app/views/layouts/layout.scala.html

```diff
@@ -31,6 +33,13 @@
     <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
     <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
     <script type="text/javascript" src='@controllers.routes.Assets.versioned("javascripts/gtm.js")'></script>
+    @hmrcTimeoutDialog(TimeoutDialog(
+        language = Some("en"),
+        timeout = Option(appConfig.timeout),
+        countdown = Option(appConfig.countdown),
+        keepAliveUrl = Some("/register-for-vat/renew-session"),
+        signOutUrl = Some(controllers.callbacks.routes.SignInOutController.signOut().url)
```

Comment:
This endpoint takes the user to feedback frontend without showing them they're signed out. What we should do is take them to a page that says: "For your security we signed you out" and "We did not save your answers". Have a look at [HMRC Design Patterns](https://design.tax.service.gov.uk/hmrc-design-patterns/service-timeout/#technical-details).

Reply:
I see where you're coming from and I'll bring this up with design tomorrow but all the previous services updated with this script have been approved in demo by design, so all previous services will need updating too if this is the case.

Reply:
Yes I agree with Nathan that this is what has been agreed with design in the short term, but what you're saying is also correct Ali, though the design work on those pages is still to be done as I understand.